ckanext/canada/tests/factories.py

```diff
@@ -0,0 +1,12 @@
+from ckan.tests.factories import Organization
+
+class Canada_Organization(Organization):
```

Comment:
```suggestion
class CanadaOrganization(Organization):
```
python style issue (don't use _ in class names)